dashboard/pages/subscriptions.py

```diff
@@ -82,19 +87,29 @@ def show_subscriptions():
             st.write(f"You are now subscribed to the {tags} tag(s)")
             for tag in tags:
                 tag = tag.replace(" ", "-")
-                if f'arn:aws:sns:eu-west-2:129033205317:c11-bandcamp-{tag}' not in topic_arns:
+                if (
```

Comment:
Could you potentially just make this a global constant? Where it's something similar to 

BASE_URL = 'arn:aws:sns:eu-west-2:129033205317:c11-bandcamp-'

Reply:
Implemented now.